sdk/eventhubs/azure-messaging-eventhubs-stress/pom.xml

```diff
@@ -0,0 +1,62 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
```

Comment:
I am sure do we need a parent dependency here. Do we need to create a common parent dependency for stress purpose, or use existing client/perf parent?

Reply:
I am not sure in the java case, but for stress tests in general it is becoming a common pattern to reference parent dependencies, though you need to pass a custom docker build directory to support it (e.g. https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azservicebus/internal/stress/Chart.yaml#L9).

Reply:
Currently I use existing `azure-client-sdk-parent` as parent here.  If we want to have a common parent for stress test to control some specific dependencies/plugins, we can create a `azure-stress-test-parent` to do that.